README.md

```diff
@@ -4,15 +4,13 @@ Consul is a service discovery tool created by [HashiCorp](https://www.consul.io/
 
 # Overview
 
-When you have multiple services in a Layer0 environment, how do you get them to connect to each other? One simple answer is that you hardcode service endpoints (i.e. a Loadbalancer URL) in your application configurations. That may be OK for testing, but ultimately hardcoding endpoints is not portable, annoying to maintain, and not in line with the [guidelines of a 12-factor app](https://12factor.net/). A much better solution is to use a service like Consul, which along with another tool called Registrator, can automatically detect your services when they come online and add them to a service registry. You can then 'discover' these services using catalog endpoints via [DNS](https://www.consul.io/docs/agent/dns.html) or [HTTP](https://www.consul.io/docs/agent/http.html).
-
 For a practical applications of Consul, see our [Layer0 Consul Documentation](http://docs.xfra.ims.io/guides/consul/).
```

Comment:
[sort of nit] This sentence doesn't make sense grammatically and the link is now out of date.

Reply:
True. Docs are still WIP, I'll get to those tmrw.